gradle-plugin/it/src/it/spring-boot-with-fragment/gradle.properties

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+jkube.imagePullPolicy=Always
```

Comment:
Isn't this test covered by the imagepullpolicy scenarios?

I think having this here might add some confusion to the test, especially because it's focused on fragments.

In case you want to test something specific, maybe an additional test case can be added to SpringBootWithFragmentIT to verify that the fragment can settings can be overridden with properties, but those properties should be specified in the CLI `withArguments`

Reply:
Let me remove this.